### PR TITLE
Feature/plant details

### PR DESCRIPTION
**Pull request by SabinePfanner**
Related user story: https://github.com/SabinePfanner/plants-app/issues/3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "next": "13.4.8",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "router": "^1.3.8",
```
Comment (doemser) [inline package.json]: this should be uninstalled again:

`npm uninstall router`

File: pages/[id].js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  align-items: center;
+  padding: 10px;
+  text-decoration: none;
+  color: black;
+`;
+
+export default function DetailsPage() {
+  return (
+    <>
+      <StyledLink href="/">←</StyledLink>
```
Comment (niehaus-alexander) [inline pages/[id].js:21]: the arrow is pretty small, maybe size it up a little

File: pages/_app.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import GlobalStyle from "../styles";
 import { SWRConfig } from "swr";
+import Header from "@/components/header";
```
Comment (niehaus-alexander) [inline pages/_app.js]: By convention, React component names should be written in PascalCase (e.g., Header.js, Card.js, CardList.js). This ensures consistency with the component function names.

**Review by niehaus-alexander — CHANGES_REQUESTED** [review]
Good work, just 2 tiny things, the arrow might be something for later, not sure.

File: pages/[id].js (hunk 2 above)
Comment (niehaus-alexander) [reply pages/[id].js:21]: also i guess this is a QA thing, my bad

File: package.json (hunk 1 above)
Comment (SabinePfanner) [reply package.json]: Can you please clarify why, as we use "router" to fetch data from database by id? Thanks :)

File: pages/[id].js (hunk 2 above)
Comment (SabinePfanner) [reply pages/[id].js:21]: Thanks, will be handled within next sprint while doing styling.

File: package.json (hunk 1 above)
Comment (doemser) [reply package.json]: You are using this package?
https://www.npmjs.com/package/router

Can you point me to where you do that?

Comment (SabinePfanner) [reply package.json]: We're using ```const router = useRouter()``` in our component PlantDetails.js to get the plant data for the specific id.

File: pages/_app.js (hunk 3 above)
Comment (SabinePfanner) [reply pages/_app.js]: Thanks, done.

File: package.json (hunk 1 above)
Comment (SabinePfanner) [reply package.json]: Done, package is uninstalled

**Review by niehaus-alexander — APPROVED** [review]
LGTM !

**Pull request merged**